windows/container.go

```diff
@@ -174,6 +174,11 @@ func (c *container) Pid() uint32 {
 	return c.ctr.Pid()
 }
 
+func (c *container) Ps(ctx context.Context) ([]uint32, error) {
+	// TODO (ehazlett): support finding all processes in windows container
```

Comment:
You can call `ProcessList()` on the `hcsshim.Container` object. It is embedded in the `windows/hcs/Container` object.

Be careful though, on windows there's currently not a 1 to 1 mapping between the `pids` returned by the runtime and the one of the system due to the fact that the `pid` is currently needed a container creation but we only get the init pid at container startup with the `hcsshim`. 

And as of now, I am not sure how to best fix this situation.